File: abc/abc207/a.java
// https://atcoder.jp/contests/abc207/submissions/40778087

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);
    
    int a = stdIn.nextInt();
    int b = stdIn.nextInt();
    int c = stdIn.nextInt();
    
    System.out.println(Math.max(a + b, Math.max(b + c, c + a)));
  }
}

File: abc/abc207/a.rs
// https://atcoder.jp/contests/abc207/submissions/47813133

use proconio::input;

fn max(a: u16, b: u16) -> u16 {
    if a > b {a} else {b}
}

fn main() {
    input! {
        a: u16,
        b: u16,
        c: u16
    }

    print!("{}", max(a + b, max(b + c, c + a)));
}

